install/Knative-with-AKS.md

```diff
@@ -110,7 +110,7 @@ recommended configuration for a cluster is:
    az aks create --resource-group $RESOURCE_GROUP \
    --name $CLUSTER_NAME \
    --generate-ssh-keys \
-   --kubernetes-version 1.10.5 \
+   --kubernetes-version 1.11.5 \
```

Comment:
Might be safer to recommend 1.11.0? I'd like someone from AKS to confirm what version they'd like to see here.

Reply:
@jeremyrickard , who I believe wrote this guide initially

Reply:
Oh, perfect!